README.md

```diff
@@ -48,6 +48,11 @@ Table of Contents
     * [Sending Email](#sending-email)
         * [Mailgun](#mailgun)
     * [Localization](#localization)
+    * [Interceptors](#interceptors)
+        * [DataRequiredInterceptor](#datarequiredinterceptor)
+            * [Configuration](#configuration)
```

Comment:
Please only go three levels deep here, to keep the TOC shorter.   (keep the sections, just don't list them here beyond three levels. Thanks!)